src/index.js

```diff
@@ -226,26 +226,37 @@ VirtualMachine.prototype.fromJSON = function (json, testing) {
     }
 };
 
-VirtualMachine.prototype.testJSON = function () {
-    console.log('Exporting to JSON...');
-    var json = this.toPrettyJSON(true);
-    console.log('Importing from JSON...');
-    this.fromJSON(json, true);
-    console.log('Exporting to JSON...');
-    var json2 = this.toPrettyJSON(true);
+VirtualMachine.prototype.testJSON = function (debug) {
+    if (debug == true) {
+        console.log('Exporting to JSON...');
+    }
```

Comment:
A shorter way to do this would be something like this:

```js
if (debug) console.log(...)
```

Or you could make a "debug" function:

```js
var deb = function() {
  if (debug == true) console.log.apply(console, arguments)
}

deb(...)
```